handler.go

```diff
@@ -875,6 +882,20 @@ func (h *Handler) readURLQueryRequest(r *http.Request) (*QueryRequest, error) {
 	}, nil
 }
 
+func (h *Handler) getValidURLQuery(r *http.Request) map[string]bool {
```

Comment:
Since this function doesn't depend on a Handler, would it make sense to make it a standalone function? Also name of the function is a bit misleading. Would something like `validURLQueryArgs` work better ?